src/mono/Directory.Build.props

```diff
@@ -13,11 +13,11 @@
 
   <PropertyGroup>
     <!-- Minimum target OS version, keep in sync with src/native/libs/build-native.sh -->
-    <iOSVersionMin>10.0</iOSVersionMin>
-    <tvOSVersionMin>10.0</tvOSVersionMin>
+    <iOSVersionMin>11.0</iOSVersionMin>
```

Comment:
nit: should this move level up because of nativeaot?

Reply:
what do you mean? this is unrelated to nativeaot.

Reply:
I know just commenting on the code as you touch it. I think it'd be better under src/

Reply:
ah, yeah that might be a good idea

Reply:
Done